### ISSUE TITLE
Pathfinding when no path

### ISSUE THREAD

**Issue by ErezBinyamin** [pran01/AlgoVision#13]
When in the path-finding tab, it is possible that no path exists. Some message or effect should be shown when this happens.

**Comment by pran01** [comment]
Yeah currently if theres no path, it doesn't show any path. Thats it. But, I guess I can add an alert or modal saying no path exists. I will see to it. Thanks for feedback
